client/src/components/ApplicationReviewForm/ApplicationReviewForm.tsx

```diff
@@ -0,0 +1,205 @@
+import { useApplicationsContext } from '../../contexts/ApplicationsProvider/hooks'
+import { isNotEmpty, useForm } from '@mantine/form'
+import { GLOBAL_CONFIG } from '../../config/global'
+import React, { useEffect, useState } from 'react'
+import { useDebouncedValue } from '@mantine/hooks'
+import { doRequest } from '../../requests/request'
+import { Button, Checkbox, Divider, Group, Stack, Text, Textarea, TextInput } from '@mantine/core'
+import UserMultiSelect from '../UserMultiSelect/UserMultiSelect'
+import { isNotEmptyUserList } from '../../utils/validation'
+import { showSimpleError, showSimpleSuccess } from '../../utils/notification'
+
+interface IApplicationReviewFormProps {
+  application: IApplication
+  onUpdate: () => unknown
+}
+
+const ApplicationReviewForm = (props: IApplicationReviewFormProps) => {
+  const { application, onUpdate } = props
+
+  const { updateApplication } = useApplicationsContext()
+
+  const form = useForm<{
+    title: string
+    comment: string
+    advisors: string[]
+    supervisors: string[]
+    notifyUser: boolean
+  }>({
+    mode: 'controlled',
+    initialValues: {
+      title: application?.thesisTitle || '',
+      comment: '',
+      advisors: [],
+      supervisors: GLOBAL_CONFIG.default_supervisors,
+      notifyUser: true,
+    },
+    validateInputOnBlur: true,
+    validate: {
+      title: isNotEmpty('Title must not be empty'),
+      advisors: isNotEmptyUserList('advisor'),
+      supervisors: isNotEmptyUserList('supervisor'),
+    },
+  })
+
+  useEffect(() => {
+    form.reset()
+    form.setFieldValue('title', application?.thesisTitle || '')
+    form.setFieldValue('comment', application?.comment || '')
+  }, [application?.applicationId])
+
+  const [loading, setLoading] = useState(false)
+
+  const [debouncedComment] = useDebouncedValue(form.values.comment, 1000)
+
+  useEffect(() => {
+    if (application && debouncedComment !== application.comment) {
+      doRequest<IApplication>(
+        `/v2/applications/${application.applicationId}/comment`,
+        {
+          method: 'PUT',
+          requiresAuth: true,
+          data: {
+            comment: debouncedComment,
+          },
+        },
+        (res) => {
+          if (res.ok) {
+            application.comment = res.data.comment
+
+            updateApplication(res.data)
+          }
+        },
+      )
+    }
+  }, [debouncedComment, application?.applicationId])
+
+  return (
+    <form>
+      {application?.state === ApplicationState.NOT_ASSESSED && (
+        <Stack gap='md'>
+          <Divider my='md' />
+
+          <TextInput
+            type='text'
+            required={true}
+            placeholder='Thesis Title'
+            label='Thesis Title'
+            {...form.getInputProps('title')}
+          />
+
+          <UserMultiSelect
+            label='Supervisor'
+            required={true}
+            groups={['supervisor']}
+            maxValues={1}
```

Comment:
Its a requirement from Stephan to only have 1 supervisor selectable. Without this line multiple would be supported so it can be changed easily when requirements would change